packages/dashboard/src/app/views/exploreTemplates/content/templateGridItem.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@web-stories-wp/i18n';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+} from '@web-stories-wp/design-system';
+import { forwardRef } from '@web-stories-wp/react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { TEMPLATES_GALLERY_ITEM_CENTER_ACTION_LABELS } from '../../../../constants';
+import { CardGridItem } from '../../../../components';
+import { Container, Gradient, Scrim } from '../../shared/grid/components';
+import {
+  TemplateDisplayContent,
+  CardWrapper,
+  PicturePoster,
+} from './components';
+
+export const FOCUS_TEMPLATE_CLASS = 'focus_template';
+
+const TemplateGridItem = forwardRef(
+  (
+    {
+      detailLink,
+      handleCreateStory,
+      handleFocus,
+      height,
+      id,
+      isActive,
+      posterSrc,
+      slug,
+      title,
+      status,
```

Comment:
Nit: do we want to spread any extra props into The `CardGridItem` or `Container` (such as `className`)?

Reply:
right now, i'd rather not. There's also #8652 in flight that touches that code. Getting all of the grids on the same page will make this easier to clean up. For now, I'm just trying to focus on swapping out the template structure for images. It should clear a path for us to do a good bit of clean up in a more straight forward manner.